live_data/mqtt_subscriber/oli_broker.py

```diff
@@ -47,7 +47,10 @@ def on_message(self, client, userdata, msg):
             energy = payload["value"]
 
             # Transmit power values to CN
+            print(f"self.topic_api_client_dict: {self.topic_api_client_dict}")
+            print(f"msg.topic: {msg.topic}")
```

Comment:
Please either convert these to debug logs or remove these before merging if you do not need them anymore